src/MaxText/configs/models/gpu/mixtral_8x7b.yml

```diff
@@ -30,6 +30,6 @@ reuse_example_batch: 1
 enable_checkpointing: False
 megablox: False
 scan_layers: False
-tokenizer_path: "/deps/assets/tokenizer.mistral-v1"
+tokenizer_path: "/deps/src/MaxText/assets/tokenizer.mistral-v1"
```

Comment:
Not for this PR since it was already set to this. But @parambole is `/deps` the right prefix here? Or will this only work with Docker?